docs/infrastructure.md

```diff
@@ -142,8 +142,8 @@ To recreate an inactivated app, or create one for a PR from a fork, you can use
 
 ### Rolling back deployments
 
-Normal Heroku deployments, performed by our GitHub Actions CI workflow, take about 6 minutes.
-Heroku allows us to immediately return to a previous version using rollbacks.
+In case of issues with a deployment, Heroku allows us to immediately return to a previous version using rollbacks, which take only several seconds to take effect.
+Normal Heroku deployments, performed by our GitHub Actions CI workflow, take about 6 minutes, and so "failing forward" with a reversion in Git and new deploy, if appropriate, should be done secondary to a rollback for time-sensitive issues.
```

Comment:
_very nit, much pick_

initially, i was going to suggest this: 
```suggestion
Normal Heroku deployments, performed by our GitHub Actions CI workflow, take about 6 minutes, and so "failing forward" with a reversion in Git and new deploy, if appropriate, should be done secondarily to a rollback for time-sensitive issues.
```

...but I don't think that reads any better, or is any clearer, so instead I suggest:

```suggestion
Normal Heroku deployments, performed by our GitHub Actions CI workflow, take about 6 minutes, and so, for time-sensitive issues, "failing forward" with a reversion in Git and new deploy, if appropriate, should be done after a rollback.
```

Reply:
Sure, applied. (I admit I didn't take too much time re-editing my Slack post that this language came from.)